### ISSUE TITLE
[Feature request] Headscale support

### ISSUE THREAD

**Issue by Twig6943** [f-koehler/KTailctl#319]
Not sure if this is a thing. (Doesn't seem to be)

**Comment by f-koehler** [comment]
Yes, headscale support would be nice (I was hoping for it to just work out of the box. What does happen at the moment? Does KTailctl just crash? Or display no information?

**Comment by code8buster** [comment]
Running this with headscale works for the limited set of features i've tested. Certain things that are specific to tailscale just dont work i.e. mullvad exits, exit node suggestions, etc.

You can see and use peers that are running exits. That works beautifully.

**Comment by f-koehler** [comment]
@code8buster Thanks for the report. Good to hear that the core functionality works correctly! Sounds like this will be more about disabling some GUI elements that link to features that are not supported by Headscale.
